app/models/proposal.rb

```diff
@@ -13,7 +13,7 @@ class Proposal < ActiveRecord::Base
 
   # The following list also servers as an interface spec for client_datas
   delegate :fields_for_display, :client, :public_identifier, :total_price,
-           :name, to: :client_data_legacy
+           :name, :version, to: :client_data_legacy
```

Comment:
That ties it down to comparable version identifiers, which is okay for the time being. Will implement.